### DISCUSSION TITLE
Problem with abr (Origin-Edge)

### DISCUSSION THREAD

**Discussion by cloudcam-svtech** [OvenMediaLabs/OvenMediaEngine#1311] [Help] [answered]
I'm running a origin-edge model with one origin and one edge. The Origin receives streams (about 400 - 500 camera streams) from a nimble streaming server with the same app and use OVT publisher to Edge. At Edge, I publish LLHLS/WEBRTC, everything works fine. However, it throws error "stream not found" when I configure Edge to encode abr and play the stream with ovenplayer.
![image](https://github.com/AirenSoft/OvenMediaEngine/assets/107380055/d4945e53-10e9-46d9-8d61-db5377702f32)

Then I create another app "test" on both Origin and Edge, use OBS to stream to Origin and config abr on Edge. It works like a charm.
I don't know if the too many streams use the same app is the problem
![image](https://github.com/AirenSoft/OvenMediaEngine/assets/107380055/ad648076-7d0a-4494-95bc-3fe1d2e355dd)

OME version: v0.15.13 (Origin & Edge)
Oven Player: v0.10.30

**Comment by getroot** [comment, marked as answer]
Encoding and playlist configuration must be in Origin. Streams received over the OVT bypass the encoder. Check the log.

**Reply by cloudcam-svtech** [reply]
Thanks for your quick answer. Actually, it was my misunderstanding, I had configured encoding the app "test" in both Origin and Edge, therefore it worked. Is there a way to configure encode in Edge to reduce resources consumption? Unlike the Origin, the Edge will only encode the necessary streams.
